infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/bootstrapper/OpenShiftBootstrapper.java

```diff
@@ -114,14 +114,16 @@ protected void doBootstrapAsync(String installerWebsocketEndpoint, String output
   }
 
   private void injectBootstrapper() throws InfrastructureException {
-    LOG.debug("Creating folder for bootstrapper");
+    LOG.debug(
+        "Bootstrapping {}:{}. Creating folder for bootstrapper", runtimeIdentity, openShiftMachine);
     openShiftMachine.exec("mkdir", "-p", BOOTSTRAPPER_DIR);
-    LOG.debug("Downloading bootstrapper binary");
+    LOG.debug(
+        "Bootstrapping {}:{}. Downloading bootstrapper binary", runtimeIdentity, openShiftMachine);
```

Comment:
openShiftMachine is already logged in line 117. We can avoid it here. WDYT?

Reply:
I think OpenShift machine name can be useful here. Because infrastructure can implement bootstrapping in a parallel way, then logging won't be sequential and it won't be clear to which machine these logs belong to. Does it make sense?

Reply:
Yes, with the name it's OK, I was concerned about the whole object